src/v2/client/startLoginFlow.js

```diff
@@ -22,24 +23,47 @@ const handleProxyIdxResponse = async (settings) => {
   });
 };
 
-export async function startLoginFlow(settings, appState) {
+export async function startLoginFlow(settings) {
   // Return a preset response
   if (settings.get('proxyIdxResponse')) {
     return handleProxyIdxResponse(settings);
   }
 
-  // Use stateToken
-  const stateHandle = settings.get('stateToken');
-  if (stateHandle) {
-    return introspect(settings, appState);
+  if (settings.get('overrideExistingToken')) {
+    sessionStorageHelper.removeStateHandle();
   }
-  
+
   // Use or acquire interactionHandle
   const useInteractionHandle = settings.get('useInteractionCodeFlow') || settings.get('interactionHandle');
   if (useInteractionHandle) {
     return interact(settings);
   }
 
-  throw new Errors.ConfigError('Set "useInteractionCodeFlow" to true in configuration to enable the '+ 
+  // Use stateToken from session storage if exists
+  // See more details at ./docs/use-session-token-prior-to-settings.png
+  const stateHandleFromSession = sessionStorageHelper.getStateHandle();
```

Comment:
Should we consider a fallback to the model `stateToken` if this doesn't return a value?

Reply:
when `stateHandleFromSession` is null, it continues to next block where introspect on `setting.stateToken`.
Is this fallback you're wondering?